### ISSUE TITLE
Add a directive for loading `ini`files

### ISSUE THREAD

**Issue by jakubtobiasz** [dunglas/frankenphp#1229]
### Describe you feature request

**Is your feature request related to a problem? Please describe.**
I have the following configuration:
```
{
	frankenphp
	order php_server before file_server
	log {
        level DEBUG
    }
}

:443 {
	tls /cert/_wildcard.orb.local.pem /cert/_wildcard.orb.local-key.pem

	root * {$PUBLIC_DIR:/app/public}

	@has_xdebug_cookie header Cookie *XDEBUG_SESSION*

	route @has_xdebug_cookie {
		php_server {
			env XDEBUG_MODE "debug" <---- it doesn't work here :(
		}
	}

	route {
		php_server 
	}
}
```

To improve performance while developing an app, I'd like to provide a conditional xdebug enabling. I've noticed the XDebug Helper sets the `XDEBUG_SESSION` cookie. So, I know when I should do *something* (enable XDebug). However, I've noticed the only way to provide an additional `.ini` file is by setting an environment variable, which is not possible for me. I'm not sure if it is possible, but it would be great if I could provide an extra `.ini` file as a directive instead of an environment variable.

**Describe the solution you'd like**
Something like:
```
...
	route @has_xdebug_cookie {
		php_server {
			ini "some_path_to_my.ini"
		}
	}
...
```

**Describe alternatives you've considered**
n/a

**Comment by dunglas** [comment]
Good idea. We could even support setting INI directives directly in the `Caddyfile`.

**Comment by henderkes** [comment]
we have php_ini in Caddyfile now. I'm not sure loading extra ini files is worth it outside of phps mechanisms. I'd prefer to keep things simple, can always override PHP_INI_SCAN_DIR or add to PHPRC

**Comment by henderkes** [comment]
(fyi the approach won't work for conditionally enabling debug. xdebug must be enabled on server startup and can't be controlled per-php_server. I recommend running fpm or a second frankenphp to proxy with xdebug instead)
